### ISSUE TITLE
[Vanilla Bug] Building an MCV out of a captured factory produces home MCV

### ISSUE THREAD

**Issue by hacklex** [Vinifera-Developers/Vinifera#939]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I have attached as much information as possible *(screenshots, debug and exception logs, etc)*.

### Description

In mods, DTA in particular, there are different MCVs for different factions, but should I build another house's MCV (after capturing enemy one), the game will not treat properly and upon deploying it I won't get access to that side's techs.

Probably, one of the approaches would be to have the MCV inherit the faction flag of the factory it was constructed in...

### Steps To Reproduce

In DTA (or supposedly any other mod that has different MCVs for different factions), 

capture a factory and a tech center (MCV prerequisites) from the enemy, and build the enemy mcv (which is a different unit from your own one)

### Expected Behaviour

We would naturally expect to have access to the enemy tech tree

### Actual Behaviour

Deployed MCV looks like the enemy con yard, but does not provide enemy tech.

**Comment by Bittah** [comment]
The same issue exists in Vanilla TS when you capture a Nod War Factory as GDI and produce an MCV from it or capture a GDI War Factory as Nod and produce an MCV from that.

**Comment by hacklex** [comment]
yep. For example, in a RA1 mission when you capture enemy base and then make a war factory, this war factory will give you your own units.
